pages/k8s/how-to-cos-lite.md

```diff
@@ -96,6 +96,13 @@ you can run `juju models` to see a list of available models):
 juju switch <charmed-kubernetes-model>
 ```
 
+Ensure `tokens` is related to all worker applications
+
+```
+juju integrate kubernetes-control-plane:tokens kubernetes-worker
+... repeat for other worker applications
```

Comment:
We generally try to keep instructions our of code blocks as many people just copy and paste. Can we move the ... repeat bit out of here?